src/styles/components/offer-details.scss

```diff
@@ -29,55 +69,18 @@
         }
 
         & > .thumb:hover {
```

Comment:
Can't accept a selector like this:
`.main-box__carousel>div:nth-child(2)>div>ul>.thumb.selected`

Style it with just 1 nested level instead of 4.